actions/chats.js

```diff
@@ -40,10 +48,10 @@ const createChat = (data) => {
     };
 };
 
-export const createSocketConnection = (data) => {
+export const createSocketConnection = () => {
     return {
         type: types.CREATE_SOCKET_CONNECTION,
-        payload: data
+        payload: io(baseUrl.slice(0, baseUrl.search('api')))
```

Comment:
Initialize socket connection. This is called in componentDidUpdate on the ChatsScreen.